files/es/mdn/writing_guidelines/howto/markdown_in_mdn/index.md

```diff
@@ -1,35 +1,592 @@
 
-- Refetencia CSS
-  - Una [referencia exhaustiva](/es/docs/Web/CSS/Reference) para _desarrolladores Web expertos_ describiendo cada propiedad y concepto de CSS.
-- Tutorial CSS
-  - Una [introduccion paso a paso](/es/docs/Learn/CSS/First_steps) para ayudar a los _principiantes_. Presenta todos los fundamentos necesarios.
-- CSS3 Demos
-  - A [collection of demos](/es/docs/Web/Demos) showing the _latest CSS technologies_ in action: a boost for the creativity.
+## Enlaces
 
-## Dos columnas
+La especificación de GFM define dos tipos básicos de enlaces:
 
-Dos columnas separadas con un borde gris. Normalmente usadas en páginas de destino. Clase de CSS: `.topicpage-table` (L 830 - 837 & 82-93 in CustomCSS).
+- [enlaces en línea](https://github.github.com/gfm/#inline-link), en los que el destino se proporciona inmediatamente después del texto del enlace
+- [enlaces de referencia](https://github.github.com/gfm/#reference-link), en los que el destino se define en otra parte del documento.
 
-Columna 1
+En MDN solo permitimos enlaces en línea.
 
-Columna 2
+Esta es la forma correcta de escribir enlaces de GFM en MDN:
 
-## Columnas con la misma altura
+```md example-good
+Los [macarrones](<https://es.wikipedia.org/wiki/Macarrón_(galleta)>) son deliciosos pero difíciles de hacer.
+```
 
-Se usa en la página de destino de [Firefox OS](/es/Firefox_OS) para envolver las columnas que deberían tener el mismo alto. Clase de CSS: `.equalColumnHeights` (L 25 - 38 in CustomCSS).
+Esta es una forma incorrecta de escribir enlaces en MDN:
 
-<div class="zone-callout">Un Texo<br>y nueva linea<br></div>
+```md example-bad
+Los [macarrones][macaron] son deliciosos pero difíciles de hacer.
 
-<div class="zone-callout">Otro texto<br><br>aquí</div>
+[macaron]: https://es.wikipedia.org/wiki/Macarrón_(galleta)
+```
+
+## Bloques de código de ejemplo
+
+En GFM y CommonMark, los autores pueden usar "vallas de código" para demarcar bloques `<pre>`. El código de apertura puede ir seguido de algún texto llamado "cadena de información". La especificación establece lo siguiente:
+
+> La primera palabra de la cadena de información se usa normalmente para especificar el idioma del ejemplo de código y se representa en el atributo de clase de la etiqueta de código.
+
+Está permitido que la cadena de información contenga varias palabras, como:
+
+````md
+```fee fi fo fum
+// algún código de ejemplo
+```
+````
+
+En MDN, los escritores usarán vallas de código, por ejemplo, bloques de código. They must specify the language of the code sample using the first word of the info string, and this will be used to provide syntax highlighting for the block. The following words are supported:
```

Comment:
Ya que puede ser un termino confuso, dejamos el concepto principal

```suggestion
En MDN, los escritores usarán bloques de código. They must specify the language of the code sample using the first word of the info string, and this will be used to provide syntax highlighting for the block. The following words are supported:
```

Reply:
se me habrá colado esto en inglés